setup.py

```diff
@@ -437,8 +437,29 @@ def find_tbb(tbb_root=None, tbb_include=None, tbb_link=None):
             long_description=readme,
             long_description_content_type='text/x-rst',
             url='https://github.com/glotzerlab/freud',
+            download_url='https://pypi.org/project/freud-analysis/',
+            keywords=('simulation analysis molecular dynamics soft matter'
+                      'particle system computational physics'),
+            classifiers=[
+                "Development Status :: 5 - Production/Stable",
+                "Intended Audience :: Science/Research",
+                "License :: OSI Approved :: BSD License",
+                "Topic :: Scientific/Engineering :: Chemistry",
+                "Topic :: Scientific/Engineering :: Physics",
+                "Programming Language :: C++",
+                "Programming Language :: Cython",
+                "Programming Language :: Python :: 3.5",
+                "Programming Language :: Python :: 3.6",
+                "Programming Language :: Python :: 3.7",
+                "Programming Language :: Python :: 3.8",
+            ],
             packages=['freud'],
             zip_safe=False,
+            maintainer='freud Developers',
+            maintainer_email='freud-developers@umich.edu',
+            # See documentation credits for current and former lead developers
+            author='Vyas Ramasubramani et al.',
+            author_email='vramasub@umich.edu',
```

Comment:
@vyasr I think you're the right person to list first since you're first author on the primary paper and have the most contributions by both lines and commits. If anyone has other ideas, input is welcome.